helm/Chart.yaml

```diff
@@ -8,10 +8,6 @@ home: https://github.com/inhindsight
 sources:
   - https://github.com/inhindsight/hindsight
 dependencies:
-  - name: redis
```

Comment:
do we want to leave this in and just set it to `enabled: false`?

Reply:
The AC said rip out Redis, we aren't using it any more. Having two stores doesn't make sense.